docs/website/01_framework/observer.md

```diff
@@ -85,13 +92,17 @@ unSubscribe(...) // (4)!
 To send a message together with data, the sender (observable) calls
 
 ```cpp
-this->notify(MessageType::Message, data);
+this->notify(MessageType message, const InfoType& info);
```

Comment:
This code is at callside, therefore there should be now type information. If you want to indicate it you can make more lines wher you create the objects. 

```cpp

InfoType info;
this->notify(message,  info);
```
or similar